### PR TITLE
Allow dealerdirect/phpcodesniffer-composer-installer

### PR DESCRIPTION
**Pull request by wojsmol**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,6 +17,11 @@
         "phpunit/phpunit": "^6.5.5 || ^7.0.0",
         "wp-coding-standards/wpcs": "^0.14.0 || ^1.0.0 || ^2.0.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1"
+    },
+	"config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        },
     },
```
Comment (schlessera) [inline composer.json:25]:
```suggestion
    "config": {
        "allow-plugins": {
            "dealerdirect/phpcodesniffer-composer-installer": true
        }
    },
```

**Review by schlessera — CHANGES_REQUESTED** [review]
The schema does not validate.

File: composer.json (hunk 1 above)
Comment (wojsmol) [reply composer.json]: fixed in https://github.com/wp-cli/wp-config-transformer/commit/8854552236c8838ea3caa4f6a62c5901af84543b

**Comment by wojsmol** [comment]
@schlessera There is strange issue https://github.com/wp-cli/wp-config-transformer/runs/4763441210?check_suite_focus=true

File: composer.json [hunk 2]
```diff
@@ -17,6 +17,11 @@
         "phpunit/phpunit": "^6.5.5 || ^7.0.0",
         "wp-coding-standards/wpcs": "^0.14.0 || ^1.0.0 || ^2.0.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1"
+    },
+	"config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        }
```
Comment (swissspidy) [inline composer.json:24]: Indentation is incorrect

**Comment by wojsmol** [comment]
This tests failures are IMHO related to to l;ow php version.

**Comment by schlessera** [comment]
@wojsmol You can rebase against latest `master` now to have the failing tests fixed.

**Comment by wojsmol** [comment]
@schlessera Done via Github UI.

**Comment by schlessera** [comment]
Sorry, @wojsmol, I just noticed that Composer had already added the `allow-plugins` section when I did the changes. This PR now adds a redundant second `config` section. I'm closing this one, as it is not needed anymore.

**Pull request closed without merging**